src/components/RCLink/RCLink.tsx

```diff
@@ -46,6 +48,10 @@ const RCLink = ({
           },
         }
       }
+      // TODO: These were added in accessibility QA. Investigate adding these to the DS Link component.
+      role="link"
+      aria-disabled={disabled}
+      tabIndex={disabled ? -1 : 0}
```

Comment:
Check the QA doc. Clare wrote "The simplest solution is to remove the href to disable the link." I asked whether href should be completely removed or if it should be set to an empty string. She said the former, so do that instead of adding `aria-disabled` or `tabIndex`. `role="link"` should also already be a default of the anchor element so it may be redundant in this case.

Reply:
Sorry should have mentioned that I asked Clare about it on slack and she proposed this as an alternative to removing the href.

Reply:
@EdwinGuzman Do you think removing the href from an a tag is advisable? I've seen warnings related to this which is why I was looking for a different solution, although it would be a lot simpler than what I did here.

Reply:
Alright! No problem, can you make a note of that either in the ticket or in the document? The updated proposal seems fine but how it came about can get lost if it's not indicated somewhere.

The warnings are mostly from eslint and react-a11y right? It's "competing" solutions that are both right but removing the "href" may seem less conventional. Let's stick with this if Clare proposed it and it works just fine.

Did she also suggest toggling between tabIndex of -1 and 0? That seems like it should be -1 and null but ask her to go over this current implementation to find out.

Reply:
Actually after taking a second look, I'm not seeing any warnings from react-a11y or eslint when the href isn't present. Not certain where I ran into this problem previously, might have been due to the lack of href making links inaccessible to screen readers, which I think is what Clare wants in this case.

I think making the href null should be ok in this case since it'll keep the code simpler and it's what Clare is recommending.

Reply:
Confirmed with @EdwinGuzman that we can keep the solution that Claire proposed via our private slack (adding tabindex -1 and aria-disabled). I've updated the QA doc and the ticket with a brief write up on our conversation.